pkg/reconciler/pipelinerun/resources/pipelinerunstate_test.go

```diff
@@ -2381,6 +2381,15 @@ spec:
 		}},
 		TaskRef: &v1beta1.TaskRef{Name: "unit-test-task"},
 	}
+	unscheduledPipelineTask := v1beta1.PipelineTask{
+		Name: "unit-test-2",
+		WhenExpressions: []v1beta1.WhenExpression{{
```

Comment:
What is the significance of `whenExpression` in the test?

Reply:
None - I just copy-pasted the task above.

Reply:
will be great to avoid such copy/paste specially given that it does not bring any value to the test, thanks 🙏